spec/models/visit_spec.rb

```diff
@@ -1,6 +1,12 @@
 require 'rails_helper'
 
-RSpec.describe Visit do
+RSpec.describe Visit, type: :model do
+  subject { build(:visit) }
+
+  specify do
+    expect(subject.valid?).to be_truthy
```

Comment:
Is there any point to this, given that we already have `FactoryGirl.lint`? (It might be worth adding `factory_girl:lint` to the default Rake task, though.)